alfresco-health-processor-platform/build.gradle

```diff
@@ -26,18 +34,40 @@ dependencies {
     testImplementation 'org.slf4j:slf4j-simple:1.7.25'
 }
 
+import com.github.jengelman.gradle.plugins.shadow.tasks.ConfigureShadowRelocation
+
+task relocateShadowJar(type: ConfigureShadowRelocation) {
```

Comment:
this is automatically relocating all dependencies, is that what you want? Otherwise you could use `relocate` on the shadowJar task could be more selective.

Reply:
This was done by intention. I thought it doesn't really make sense to only relocate some of the dependencies, since all the embedded dependencies can possibly cause conflicts?